src/core/messenger.ts

```diff
@@ -1,3 +1,4 @@
+import { reportError } from '../utils/report-error'; // Keep this import relative otherwise Frontend will not be able to import this module. TODO: move all imports to relative imports
```

Comment:
This was a surprising issue. If Frontend imports _any_ module from commercial that uses a Webpack alias the build will fail as it can't resolve the alias.

As a follow up we should:
- remove all webpack aliases and use relative imports everywhere
- tidy up the export from `core/index.ts` as there are lots exported modules there that are not actually used by any consumers and are a hangover from when core was its own separate lib